08_deploy_bmo.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/bash
+
+source bmo_install_prereqs.sh
+# Get the latest bits for baremetal-operator
+go get github.com/metalkube/baremetal-operator
+pushd ~/go/src/github.com/metalkube/baremetal-operator
+
+# Start deploying on the new cluster
+
+oc --as system:admin --config ocp/auth/kubeconfig apply -f deploy/service_account.yaml
+oc --as system:admin --config ocp/auth/kubeconfig apply -f deploy/role.yaml
+oc --as system:admin --config ocp/auth/kubeconfig apply -f deploy/role_binding.yaml
+oc --as system:admin --config ocp/auth/kubeconfig apply -f deploy/crds/metalkube_v1alpha1_baremetalhost_crd.yaml
+
+export OPERATOR_NAME=baremetal-operator
+operator-sdk up local --namespace=bmo-project
+
+oc --config ocp/auth/kubeconfig apply -f deploy/crds/metalkube_v1alpha1_baremetalhost_cr.yaml
```

Comment:
This creates the example baremetal host, and we won't want to do that in all environments. Everyone is going to need to set up their own host objects to reflect the information about the VMs they have. We can automate doing that in a later patch.